### PR TITLE
Toc responsive

### PR DESCRIPTION
**Pull request by MihaelaCretu11**

### CODE REVIEW

**Comment by MihaelaCretu11** [comment]
How the Table of Contents block worked before:
before
How the Table of Contents block works now:
after

**Comment by stevepiercy** [comment]
This is so good, that it should be added to the User Manual.

https://6.docs.plone.org/volto/user-manual/blocks.html#table-of-contents-block

If possible, please see https://6.docs.plone.org/contributing/documentation/myst-reference.html#width-of-media for how to capture videos.

Please and thank you!

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Thanks for the contribution. For embedding them, please see our guide on how to make videos usable on all devices.

https://6.docs.plone.org/contributing/documentation/myst-reference.html#video

Also for all media, width is preferred to be at 740 pixels.

https://6.docs.plone.org/contributing/documentation/myst-reference.html#width-of-media

If you are unable to use a smaller size, I will take what I can get. Thank you!

**Review by stevepiercy — COMMENTED** [review]
I would like to see one of the videos, `docs/source/_static/user-manual/blocks/table-of-contents-block-with-dropdown.mp4`, embedded into https://6.docs.plone.org/volto/user-manual/blocks.html#table-of-contents-block

It looks like the other video demonstrates a bug, but I am not sure.

**Comment by stevepiercy** [comment]
There are conflicts that need to be resolved in `theme/themes/pastanaga/extras/main.less`.

File: docs/source/user-manual/blocks.md
Comment (stevepiercy) [inline docs/source/user-manual/blocks.md:512]:
```suggestion
  The horizontal variation also adds a dropdown button, if necessary, so the items that won't fit would be displayed in the dropdown list.
```

File: docs/source/user-manual/blocks.md
Comment (stevepiercy) [inline docs/source/user-manual/blocks.md:513]:
````suggestion
  ```{video} /_static/user-manual/blocks/table-of-contents-block-with-dropdown.mp4
````

File: docs/source/user-manual/blocks.md
Comment (stevepiercy) [inline docs/source/user-manual/blocks.md:514]:
````suggestion
  :alt: Table of contents block with dropdown menu.
  ```
````

**Review by stevepiercy — CHANGES_REQUESTED** [review]
This looks good. I have only minor MyST syntax corrections.

Indentation is necessary after the first colon for a definition list definition.

I also added an alt tag for the video.

You can see a preview: https://deploy-preview-4912--volto.netlify.app/user-manual/blocks.html#table-of-contents-block

**Review by stevepiercy — APPROVED** [review]
I approve docs only. A core maintainer should review the code. Thank you! This looks great!

**Review by sneridagh — APPROVED** [review]

**Comment by sneridagh** [comment]
@plone/ai-team can we add @MihaelaCretu11 to the developers team? Thanks!

**Comment by fredvd** [comment]
@sneridagh @MihaelaCretu11 Sorry, I'm messing up again.  :-$

 I wanted to add a note: I have added @MihaelaCretu11 to Developers,  by accident I modify @sneridagh 's comment, and I when I try to revert my comment I've reverted the last commit :-O

But I have no clue now how I can revert my accidental revert

**Comment by stevepiercy** [comment]
@fredvd perhaps not through GitHub UI, but try pulling the remote branch down and editing it locally, and so on?

**Comment by mauritsvanrees** [comment]
I have removed branch `revert-4912-toc-responsive`.  Seems fine now.

BTW, there are 514 branches, with some being years old.  I am sure lots could be removed.  But if this does not bother anyone, we can leave them.

**Pull request merged**
